src/data/Endpoint.js

```diff
@@ -50,7 +48,8 @@ const endpoints = {
     EXPORT_DRIVER_LIST: `${laravelUrlPrefix}/${version}/customer/driverlist/export`, // url will be updated after backend is finished
     EXPORT_ROUTE: `${laravelUrlPrefix}/${version}/customer/report/routes-export`,
     CUSTOMER_DRIVER: `${laravelUrlPrefix}/${version}/customer/driver`,
-    REQUEST_PASSWORD_RESET: `${laravelUrlPrefix}/${version}/user/password/forgot`,
+    REQUEST_PASSWORD_RESET: `${laravelUrlPrefix}/${version}/user/password/email`,
+    VALIDATE_PASSWORD_RESET_TOKEN: `${laravelUrlPrefix}/${version}/password`,
```

Comment:
URL should be /user/password/{token} according to documentation

Reply:
Will that still work? Because it's just a decorator?  http://api-test.carpal.me/v3/documentation

Reply:
@maartenhemmes but
${laravelUrlPrefix}/${version}/password

/user is also missing in above URL

Reply:
Yah, no idea why I'm getting a valid 404 response with the incorrect URL